angr/engines/concrete.py

```diff
@@ -42,30 +42,33 @@ def _check(self, state, *args, **kwargs):
 
     def _process(self, new_state, successors, *args, ** kwargs):
         # setup the concrete process and resume the execution
-        self.to_engine(new_state, kwargs['extra_stop_points'], kwargs['concretize'], kwargs['timeout'])
+        self.to_engine(new_state, kwargs['extra_stop_points'], kwargs['memory_concretize'],
+                       kwargs['register_concretize'], kwargs['timeout'])
 
         # sync angr with the current state of the concrete process using
         # the state plugin
         new_state.concrete.sync()
         successors.engine = "SimEngineConcrete"
         successors.sort = "SimEngineConcrete"
         successors.add_successor(new_state, new_state.ip, new_state.solver.true, new_state.unicorn.jumpkind)
-        successors.description = "Concrete Successors "
+        successors.description = "Concrete Successors"
         successors.processed = True
 
-    def to_engine(self, state, extra_stop_points, concretize, timeout):
+    def to_engine(self, state, extra_stop_points, memory_concretize, register_concretize, timeout):
         """
         Handle the concrete execution of the process
         This method takes care of:
         1- Set the breakpoints on the addresses provided by the user
-        2- Concretize the symbolic variables and perform the write inside the concrete process
+        2- Concretize the symbolic variables and perform the write inside the concrete process for memory and regs
         3- Continue the program execution.
 
-        :param state:               The state with which to execute
+        :param state:               The state with which to execute.
         :param extra_stop_points:   list of a addresses where to stop the concrete execution and return to the
-                                    simulated one
-        :param concretize:          list of tuples (address, symbolic variable) that are going to be written
-                                    in the concrete process memory
+                                    simulated one.
+        :param memory_concretize:   list of tuples (address, symbolic variable) that are going to be written
+                                    in the concrete process memory.
+        :param register_concretize  list of tuples (reg_name, symbolic variable) that are going to be written
```

Comment:
Is the lack of a ':' after 'register_concretize' what is making the CI fail?
```suggestion
        :param register_concretize:  list of tuples (reg_name, symbolic variable) that are going to be written
```

Reply:
Seems no, I actually forgot that this was not already merged, I'll talk with @rhelmot! Thanks for the heads up!